Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.5
+// swift-tools-version:5.6
```

Comment:
It's not building on 5.5 (in the build matrix) for some reason (didn't bother to investigate) so might as well bump it.

We _could_ also bump it to 5.7 to save the build system some time. Is it even worth having the green tick for 5.6 in the build matrix?